_includes/releases/v22.1/v22.1.12.md

```diff
@@ -0,0 +1,104 @@
+
+<h3 id="v22-1-12-db-console-changes">DB Console changes</h3>
+
+- Changed the height of column selector, so it can hint there are more options to be selected once scrolled. [#91910][#91910]
+- Added fingerprint ID in hex format to the Statement Details page and Transaction Details page. [#91959][#91959]
+- Updated tooltip on `SQL Statement Errors` chart on Metrics page. [#92711][#92711]
+
+<h3 id="v22-1-12-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug in which panics triggered by certain DDL statements were not properly recovered, leading to the cluster node crashing. [#91555][#91555]
+- Fixed a panic that could occur when calling st_distancespheroid or st_distancesphere with a spatial object containing an NaN coordinate. This now produces an error, "input is out of range". [#91634][#91634]
+- Fixed a bug that resulted in some retriable errors not being retried during IMPORT. [#90432][#90432]
+- Fixed a bug in `Concat` projection operators for arrays that could cause non-null values to be added to the array when one of the arguments was null. [#91653][#91653]
+- Fixed a bug described in #91069. [#91089][#91089]
+- Fixed a bug introduced in 21.2 that could cause an internal error in rare cases when a query required a constrained index scan to return results in order. [#91692][#91692]
+- Fixed a bug which, in rare cases, could result in a changefeed missing rows which occur around the time of a split in writing transactions which take longer than the closed timestamp target duration (defaults to 3s). [#91749][#91749]
+- Add leading zeros to fingerprint IDs with less than 16 characters. [#91959][#91959]
+- Fixed a bug introduced in 20.2 that could cause filters to be dropped from a query plan with many joins in rare cases. [#91654][#91654]
+- Fixed an unhandled error that could happen if `ALTER DEFAULT PRIVILEGES` was run on the system database. [#92083][#92083]
+- Reduced the amount that RESTORE over-splits. [#91141][#91141]
+- Fixed a bug causing changefeeds to fail when a value is deleted while running on a non-primary column family with multiple columns. [#91953][#91953]
+- Strip quotation marks from database and table names to correctly query for index usage statistics. [#92282][#92282]
+- Fixed the statement activity page so that it no longer shows multi-statement implicit transactions as "explicit." [#92430][#92430]
+- Fixed a bug existing since 20.2 that could cause incorrect results in rare cases for queries with inner joins and left joins. For the bug to occur, the left join had to be in the input of the inner join and the inner join filters had to reference both inputs of the left join, and not filter NULL values from the right input of the left join. Additionally, the right input of the left join had to contain at least one join, with one input not referenced by the left join's ON condition. [#92103][#92103]
```

Comment:
Nits:

"v20.2"
Link to the joins page on an instance: joins.html?
`NULL`
`ON`